buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/fixture/ElasticsearchFixturePlugin.groovy

```diff
@@ -39,16 +40,18 @@ class ElasticsearchFixturePlugin implements Plugin<Project> {
         }
     }
 
-    private static void createClusterFor(StandaloneRestIntegTestTask integrationTest, Project project, String version) {
+    private static void createClusterFor(TaskCollection<StandaloneRestIntegTestTask> integrationTests, Project project, String version) {
         def clustersContainer = project.extensions.getByName(TestClustersPlugin.EXTENSION_NAME) as NamedDomainObjectContainer<ElasticsearchCluster>
         def integTestCluster = clustersContainer.create("integTest") { ElasticsearchCluster cluster ->
             cluster.version = version
             cluster.testDistribution = TestDistribution.DEFAULT
         }
 
-        integrationTest.useCluster(integTestCluster)
-        // Add the cluster HTTP URI as a system property which isn't tracked as a task input
-        integrationTest.jvmArgumentProviders.add({ ["-Dtests.rest.cluster=${integTestCluster.httpSocketURI}"] } as CommandLineArgumentProvider)
+        integrationTests.all { StandaloneRestIntegTestTask integrationTest ->
```

Comment:
side note: You might want to consider moving your builds to use the task avoidance api for some better performance. see https://docs.gradle.org/current/userguide/task_configuration_avoidance.html for details